libs/accounts/src/lib/breakdown-table.spec.tsx

```diff
@@ -0,0 +1,128 @@
+import BreakdownTable from './breakdown-table';
+import { act, render, screen, waitFor } from '@testing-library/react';
+import { Schema as Types } from '@vegaprotocol/types';
+import type { AccountFields } from './accounts-data-provider';
+import { getAccountData } from './accounts-data-provider';
+
+const singleRow: AccountFields = {
+  __typename: 'Account',
+  type: Types.AccountType.ACCOUNT_TYPE_MARGIN,
+  balance: '125600000',
+  market: {
+    __typename: 'Market',
+    tradableInstrument: {
+      __typename: 'TradableInstrument',
+      instrument: {
+        __typename: 'Instrument',
+        name: 'BTCUSD Monthly (30 Jun 2022)',
+      },
+    },
+    id: '10cd0a793ad2887b340940337fa6d97a212e0e517fe8e9eab2b5ef3a38633f35',
+  },
+  asset: {
+    __typename: 'Asset',
+    id: '5cfa87844724df6069b94e4c8a6f03af21907d7bc251593d08e4251043ee9f7c',
+    symbol: 'tBTC',
+    decimals: 5,
+  },
+  available: '125600000',
+  used: '125600000',
+  deposited: '125600000',
+};
+const singleRowData = [singleRow];
+
+describe('BreakdownTable', () => {
+  it('should render successfully', async () => {
+    await act(async () => {
+      const { baseElement } = render(<BreakdownTable data={[]} />);
+      expect(baseElement).toBeTruthy();
+    });
+  });
+
+  it('should render correct columns', async () => {
+    act(async () => {
```

Comment:
chore (blocking): This is incorrect usage of `act` and there are console error warnings in test output. Please fix.

Note that these tests are also evergreen as a result.

- you must await `act(async () => ..`
- assertions should be made outside of `act` so that the component has finished rendering

Reply:
I'll fix those together with the withdraw-manager spec which is timing out.